asl/encoding/encoding.go

```diff
@@ -2,11 +2,12 @@ package encoding
 
```

Comment:
Changes in this file seem to be unnecessary

Reply:
@purplefox  some of the changes are lint/formating changes. Do we have any linter enabled in the project which I can use. Vscode seems to make these changes from it standard formatting of golang

Reply:
There are no relevant changes in this file, so can just be removed from the PR